crates/edr_napi/package.json

```diff
@@ -55,7 +55,7 @@
     "artifacts": "napi artifacts",
     "build": "pnpm run build:publish",
     "build:debug": "napi build --platform  --no-const-enum",
-    "build:dev": "napi build --platform  --release --no-const-enum",
+    "build:dev": "pnpm build:op",
```

Comment:
I changed this, because now not just the `edr_napi` tests depend on the `op` feature, but `js/integration-tests/solidity-tests` depend on it as well.

Reply:
This requires an update of `index.d.ts` for the CI to succeed.

Reply:
I updated the `index.d.ts` here: https://github.com/NomicFoundation/edr/pull/969/commits/d8fb8af48db6fc1ffbe9b94f8ad47965a1f3785d

I also revised the `edr_napi` build commands and the CI check a bit to make it more consistent:

Both `build:dev` and `build:debug` now build with feature OP for consistency in development. I removed `build:op` as I didn't see a need for it anymore.

Since `build:dev` now builds with feature OP, but in the CI we want to check that the typings file matches the released version, I added a new command `build:typingFile` that matches `build:publish` except it's unoptimized.

Reply:
Commit for changing the `edr_napi` build commands: https://github.com/NomicFoundation/edr/pull/969/commits/96d6eb6e4e9005b0ddb632dec4454d719618ceb5